### DISCUSSION TITLE
No alerts

### DISCUSSION THREAD

**Discussion by Amjadf** [Security-Onion-Solutions/securityonion#3275] [Unsupported Versions]
I have installed 2.3.30 (Latest version) on Hyper-V. I am not seen any alerts in the alerts interface other than the system itself. No error found at all. All services are working. Tried to update but didn't help. I can see logs in Kabana (26,839,424 Count logs so far).

Basic installation - Standalone

Any recommendation ?

**Comment by meredib** [comment]
Try and run "sudo so-rule-update".  I had a similar issue on a standalone kit where that resolved it.
